scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -415,9 +416,9 @@ class Framework extends SbtFramework {
     lazy val shouldDiscover = 
       taskDefinition.explicitlySpecified || ((accessible || runnable) && isDiscoverableSuite(suiteClass))
     
-    def tags = 
+    def tags = (
       for { 
-        a <- suiteClass.getAnnotations
+        a <- suiteClass.getAnnotations.map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.